### PR TITLE
Adding constants for front end to display booleans

### PR DESCRIPTION
**Pull request by emma-nechamkin**
For the most part, the variables match their front end empty variables, except one category of thresholds was missing, and all but T&WD have the same socioeconomic criteria, so that's `m_ebsi`:
```
    field_names.CLIMATE_THRESHOLD_EXCEEDED: "M_CLT_EOMI"
    field_names.ENERGY_THRESHOLD_EXCEEDED: "M_ENY_EOMI"    
    field_names.TRAFFIC_THRESHOLD_EXCEEDED: "M_TRN_EOMI"
    field_names.HOUSING_THREHSOLD_EXCEEDED: "M_HSG_EOMI"
    field_names.POLLUTION_THRESHOLD_EXCEEDED: "M_PLN_EOMI"
    field_names.WATER_THRESHOLD_EXCEEDED: "M_WTR_EOMI"
    field_names.HEALTH_THRESHOLD_EXCEEDED: "M_HLTH_EOMI"
    field_names.WORKFORCE_THRESHOLD_EXCEEDED: "M_WKFC_EOMI"
    field_names.WORKFORCE_SOCIO_INDICATORS_EXCEEDED: "M_WKFC_EBSI"
    field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES: "M_EBSI"
```

### CODE REVIEW

**Comment by emma-nechamkin** [comment]
(TBD - update fixtures when done)

**Comment by emma-nechamkin** [comment]
checked agreement with score for education and poverty line for several examples, see picture
Screen Shot 2022-03-01 at 10 33 18 AM

File: data/data-pipeline/data_pipeline/score/field_names.py [hunk 1]
```diff
@@ -59,6 +59,38 @@
...
+HOUSING_THREHSOLD_EXCEEDED = "At least one housing threshold exceeded"
+POLLUTION_THRESHOLD_EXCEEDED = "At least one pollution threshold exceeded"
+WATER_THRESHOLD_EXCEEDED = "At least one water threshold exceeded"
+HEALTH_THRESHOLD_EXCEEDED = "At least one health threshold exceeded"
+WORKFORCE_THRESHOLD_EXCEEDED = "At least one workforce threshold exceeded"
+WORKFORCE_SOCIO_INDICATORS_EXCEEDED = (
+    "Both workforce socioeconomic indicators exceeded"
+)
+
+# For now, these are not used. Will delete after following up with Vim.
```
Comment (vim-usds) [inline data/data-pipeline/data_pipeline/score/field_names.py:75]: @emma-nechamkin - I think we can remove these yes :)

**Review by vim-usds — COMMENTED** [review]
one comment

**Comment by vim-usds** [comment]
@emma-nechamkin - qq

When checking the new booleans on this ticket all of them seem great! 

When diffing the prod and staging tiles, two questions came up regarding:
1. M_EBSI
2. RMPLI

## M_EBSI
If my understanding is correct it seems that `M_EBSI` is meant to capture the _EXCEEDS BOTH SOCIAL INDICATORS_ for all categories except WFD_T. When connecting to the map, it seems to show **false** a few places even though `CA_LT20` and `FPL200S` are both true, e.g.;

Screen Shot 2022-03-01 at 3 57 48 PM

I might be misunderstanding something here. Am I using the right indicators?

## RMPLI
It seems that this indicator was being used for the boolean for the _Proximity RMP facilities_ indicator. Has this indicator's short-name changed? I think it might be missing in this build?

Screen Shot 2022-03-01 at 4 15 14 PM

Or maybe I'm using the wrong field name?

Thank you!

**Comment by emma-nechamkin** [comment]
Okay, last follow up for the night. I reran the score and these remain okay looking to me. See the variable distribution attached below. 

My tiles.csv appears correct, so I'm wondering what's different about the local generation process vs. what's on staging? 

Screen Shot 2022-03-01 at 9 49 50 PM

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**